FeedingHopeAndHeroes/FeedingHopeAndHeroes.html

```diff
@@ -0,0 +1,201 @@
+              <textarea
+                type="text"
+                class="address"
+                name="address"
+                value=""
+              /></textarea><br />
+              <label class="donation-label" for="message">Message</label><br />
+              <textarea
+                type="text"
+                class="message"
+                name="message"
+                value=""
+              /></textarea><br />
+              <button type="submit" class="donation-button form-submit-button">
+                Thank you
+              </button>
+            </div>
+          </form>
+        </div>
+        <div class="form-right">
+          <img
+            src="Images/92367a58-aa01-43f8-af3f-6e0e23c5d3fd(cropped).jpg"
+            class="doantion-form-image"
+          />
+        </div>
+      </div>
+    </div>
+
+    <div class="nhs-form-section">
+      <h1 class="nhs-form-header">
+        If you are part of the NHS and would like to talk to us
+      </h1>
+      <div class="form-container">
+        <div class="form-left">
+          <form>
+            <div class="form-col1">
+              <label class="nhs-label" for="fname">Full Name</label><br />
+              <input type="text" class="fname" name="fname" value="" /><br />
+              <label class="nhs-label" for="company">Company</label><br />
+              <input
+                type="text"
+                class="company"
+                name="company"
+                value=""
+              /><br />
+              <label class="nhs-label" for="email">Email Address</label><br />
+              <input type="text" class="email" name="email" value="" /><br />
+              <label class="nhs-label" for="contact">Contact Number</label
+              ><br />
+              <input
+                type="text"
+                class="contact"
+                name="contact"
+                value=""
+              /><br />
+            </div>
+            <div class="form-col2">
+              <label class="nhs-label" for="message">Message</label><br />
+              <textarea
+                type="text"
+                class="nhs-message"
+                name="message"
+                value=""
+              ></textarea><br />
+              <button type="submit" class="nhs-button form-submit-button">
+                Thank you
+              </button>
+            </div>
+          </form>
+        </div>
+        <div class="form-right">
+          <img
+            src="Images/8e95d135-6f51-40af-90de-3e5866b56959(croppedSquare).jpg"
+            class="nhs-form-image"
+          />
+        </div>
+      </div>
+    </div>
+
+    <div class="carousel-section">
+      <h1 class="carousel-header">
+        Follow us and share your experiences on Social Media </br>
+        @feedinghopeandheroes
+      </h1>
+      <div class="carousel-container">
+
+      </div>
+      <div class="follow-button-container">
+        <button class="follow-button">

```

Comment:
Should this be a link?

Reply:
This is also a placeholder until more details are confirmed